Cargo.toml

```diff
@@ -34,6 +34,9 @@ tzif = { version = "0.2.3", optional = true }
 jiff-tzdb = { version = "0.1.1", optional = true }
 combine = { version = "4.6.7", optional = true }
 
+# System time feature
+web-time = { version = "1.1.0", optional =  true }
```

Comment:
You've marked this as optional but isn't it required? I couldn't test this locally due to the optional being set. If it's designed to fallback to system time I assume the codepath with this will always be used (even on platforms that aren't web)

Reply:
It's optional because `Now` is optional on the `std` feature flag.